getPrivateKeys.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
```

Comment:
Can we avoid having this extra script? Its another moving part to mount, track etc.

The content of this script can be in `run_nwaku` I guess?

Reply:
TL;DR: let me see what I can do

I implemented it like this to try and simplify things because my initial implementation had everything in the run_nwaku script, but this was tricky and messy because the nwaku image runs on Alpine which can't work with arrays and indexing without installing bash, which then requires a you to execute a script to install bash and then execute the array type commands in a script which is called with the bash script and yeah, things just got messy, but let me try again and see if I can simplify it.

Reply:
The script has been removed and all logic from the script has been moved to run_nwaku.sh
Because of Alpine's default /bin/ash shell we now need to iterate through the list of private keys, which isn't ideal, but it only needs to be done once at startup so it's not too bad.